lib/migrator.js

```diff
@@ -119,6 +119,45 @@ Migrator.prototype = {
     }
   },
 
+  check: function (funcOrOpts, callback) {
+    var self = this;
+    // Not gonna lie dont really know whats up with the funcOrOpts pattern, this is just c/p'd
```

Comment:
ping on this

Reply:
Since you do not execute anything, you wont need it. The new loader is actually way more understandable:

https://github.com/db-migrate/node-db-migrate/pull/537/files#diff-69369e6549a54103e4d5c8eb543712c1

The funcOrOpts basically is an acceptor to directly execute a migration and the same time it is the enter callback for normal migration behavior.

Reply:
So just remove that bit here